CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [1.4.2]:
```

Comment:
@swasti29 is there a specific reason the new version should be 1.4.2 and not 1.4.1? The current latest version is 1.4.0, so I suppose the new version should be 1.4.1. Please, correct me if I'm wrong

Reply:
1.4.1 is the hotfix which is the latest.

Reply:
Looks like the version in changelog was updated to 1.4.1 in https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/2184/files. We typically expect developers to put TBD instead of bumping the version here when making PRs against dev branch. The release manager replaces the TBD with the actual version during release.

Other changes in the release might have warranted a major version bump so PRs against dev shouldn't assume the next version number

Reply:
Thanks, Ameya. We will make sure to always write TBD when updating the changelog file